src/main/java/org/assertj/core/error/ShouldHaveRootCause.java

```diff
@@ -14,9 +14,17 @@
 
 import static org.assertj.core.util.Objects.areEqual;
 import static org.assertj.core.util.Preconditions.checkArgument;
+import static org.assertj.core.util.Strings.escapePercent;
+import static org.assertj.core.util.Throwables.getStackTrace;
 
 public class ShouldHaveRootCause extends BasicErrorMessageFactory {
 
+  public static ErrorMessageFactory shouldHaveRootCauseWithMessage(Throwable actualCause, String expectedMessage) {
```

Comment:
Add a test for this in `ShouldHaveRootCause_create_Test`

Reply:
Done.